src/panels/media-browser/hui-dialog-web-browser-play-media.ts

```diff
@@ -76,7 +77,10 @@ export class HuiDialogWebBrowserPlayMedia extends LitElement {
               ></ha-hls-player>
             `
           : mediaType === "image"
-          ? html`<img src=${this._params.sourceUrl} />`
+          ? html`<img
+              alt=${ifDefined(this._params.title)}
```

Comment:
1. Is the title field going to be populated for image media somehow (e.g. from meta data)?
2. Can the title be an empty string? (If so I need to make a change to avoid hiding.)

I tried to follow the code trail to answer these myself but it ends at the core and docs are lacking.

Reply:
@balloob I had already pushed a change to guard against an empty string, but do you know the answer to the first question?  If core is not populating `title` for images, we should make a change to do so.

Unfortunately I don't have much in the way of media sources setup on my end, so I cannot just check that way.